### DISCUSSION TITLE
Another question about galaxy brain

### DISCUSSION THREAD

**Discussion by sourabmaity** [Schweinepriester/github-profile-achievements#102] [Q&A]
@ddh here is another question
open a discussion for me to complete galaxy brain

**Comment by masbagoel** [comment]
me too okay

**Comment by Keyrxng** [comment]
I'll take one of those

**Comment by Chameleon-Coder** [comment]
Answer!

**Comment by yllhwa** [comment]
we just need answer? I tried in my own repo, it just failed. So i'll try again
